package.json

```diff
@@ -16,6 +16,9 @@
   "dependencies": {
```

Comment:
It looks like you have two package.jsons. You'll only need one in the same root as your server.